include/tvSettingsODM.h

```diff
@@ -235,134 +149,6 @@ tvError_t GetWBctrl(char *inputSrc, char *colortemp, char *color, char *ctrl, in
- * @retval tvERROR_INVALID_PARAM    - Input parameter is invalid
- * @retval tvERROR_INVALID_STATE    - Interface is not initialized
- * @retval tvERROR_GENERAL          - Underlying failures - SoC, memory, etc
- *
- * @pre tvInit() should be called before calling this API
- */
-tvError_t ReadAllModeConfigfileODM(const char *file, char *cpybuffer, const char *searchstring);
-
-
-/**
- * @brief Gets the supported DV modes and their count.  To be deprecated soon.
- * @note SOC vendors can stub these ODM functions and mark it as weak
- *
- * This function returns the supported Dolby Vision modes and their count
- *
- * @param[out] dvModes[]            - List of available DV modes. Refer ::pic_modes_t and ::tvDolbyMode_t @n
- *                                    Valid values are "dark" and & "bright" for ::pic_modes_t.name @n
- *                                    Valid values are member of ::tvDolbyMode_t tvDolbyMode_Dark and ::tvDolbyMode_t tvDolbyMode_Bright @n
- *                                    for ::pic_modes_t.value.
- * @param[out] count                - Count of supported DV modes. Max and min is 2
- *
- * @return tvError_t
- *
- * @retval tvERROR_NONE                      - Success
- * @retval tvERROR_INVALID_PARAM             - Input parameter is invalid
- * @retval tvERROR_INVALID_STATE             - Interface is not initialized
- * @retval tvERROR_OPERATION_NOT_SUPPORTED   - Operation is not supported
- * @retval tvERROR_GENERAL                   - Underlying failures - SoC, memory, etc
- *
- * @pre tvInit() should be called before calling this API
- */
-tvError_t GetTVSupportedDolbyVisionModesODM(pic_modes_t *dvModes[],unsigned short *count);
-
-
-/**
- * @brief Sets dolby vision mode. To be deprecated soon.
- * @note SOC vendors can stub these ODM functions and mark it as weak
- *
- * This function sets the Dolby vision mode in Dolby vision core.
- * The change takes effect for current primary video source selected, primary video format if handled via Dolby vision core 
- * and picture mode selected, but not saved for future use. The value is discarded if primary video format or 
- * primary video source or picture mode changes.
- *
- * @param[in] dolbyMode              - Dolby vision mode to be set. Valid values can be one of the 
- *                                     strings from ::pic_modes_t.name parameter returned by GetTVSupportedDolbyVisionModesODM
- *
- * @return tvError_t
- *
- * @retval tvERROR_NONE                      - Success
- *
- * @retval tvERROR_INVALID_PARAM             - Input parameter is invalid
- * @retval tvERROR_INVALID_STATE             - Interface is not initialized
- * @retval tvERROR_OPERATION_NOT_SUPPORTED   - Operation is not supported
- * @retval tvERROR_GENERAL                   - Underlying failures - SoC, memory, etc
- *
- * @pre tvInit() should be called before calling this API
- *
- * @see GetTVDolbyVisionMode()
- */
-tvError_t SetTVDolbyVisionModeODM(const char * dolbyMode);
```

Comment:
SetTVDolbyVisionModeODM is expected to remove? This API is also not listed in the PR description

Reply:
Yes, it is expected to be removed. It was missing from the PR description, but I have now added it.